pkg/commands/acl/acl_test.go

```diff
@@ -10,8 +10,6 @@ import (
 	"github.com/fastly/cli/pkg/testutil"
 )
 
-const ()
```

Comment:
lol wat!

Reply:
Yeah, that was a leftover from a `sed` script which removed the only line left in the block but didn't remove the block!